server_admin/topics/users/proc-registering-new-user.adoc

```diff
@@ -0,0 +1,17 @@
+[id="proc-registering-new-user_{context}"]
+= Registering as a new user
+
+[role="_abstract"]
+As a new user you must complete a form with your profile information. 
+
+.Registration Form
+image:{project_images}/registration-form.png[]
+
+.Prerequisite
+* User registration is enabled.
+
+.Procedure
+. Click the *Register* link on the login page. The user is directed to the registration page.
```

Comment:
Can we make this "You" instead of "the user?"
Fits in with the rest of the topic.